app/components/tf-fields/label.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-ticketfly-fields/components/tf-fields/label';
```

Comment:
Weird — I swear I've done this, but maybe I've always just assumed I could, but never actually did. >.<